yasa/hypno.py

```diff
@@ -690,49 +706,49 @@ def sleep_statistics(self):
 
         Examples
         --------
-        Sleep statistics for a 2-stage hypnogram
+        Sleep statistics for a 2-stage hypnogram with a resolution of 15-seconds
 
         >>> from yasa import Hypnogram
         >>> # Generate a fake hypnogram, where "S" = Sleep, "W" = Wake
-        >>> values = 10 * ["W"] + 40 * ["S"] + 5 * ["W"] + 40 * ["S"] + 10 * ["W"]
-        >>> hyp = Hypnogram(values, n_stages=2)
+        >>> values = 10 * ["W"] + 40 * ["S"] + 5 * ["W"] + 40 * ["S"] + 9 * ["W"]
+        >>> hyp = Hypnogram(values, freq="15s", n_stages=2)
         >>> hyp.sleep_statistics()
-        {'TIB': 52.5,
-        'SPT': 42.5,
-        'WASO': 2.5,
-        'TST': 40.0,
-        'SE': 76.1905,
+        {'TIB': 26.0,
+        'SPT': 21.25,
+        'WASO': 1.25,
+        'TST': 20.0,
+        'SE': 76.9231,
         'SME': 94.1176,
         'SFI': 1.5,
-        'SOL': 5.0,
-        'SOL_5min': 5.0,
-        'WAKE': 12.5}
+        'SOL': 2.5,
+        'SOL_5min': 2.5,
+        'WAKE': 6.0}
 
-        Sleep statistics for a 5-stages hypnogram, where each epoch is one minute
+        Sleep statistics for a 5-stages hypnogram
 
-        >>> values = (10 * ["W"] + 4 * ["N1"] + 1 * ["W"] + 30 * ["N2"] + 30 * ["N3"] + 5 * ["W"]
-        ...           + 25 * ["REM"] + 15 * ["N2"] + 10 * ["W"])
-        >>> hyp = Hypnogram(values, freq="1min", n_stages=5)
+        >>> from yasa import simulate_hypno
+        >>> # Generate a 8 hr (=480 minutes) 5-stages hypnogram with a 30-seconds resolution
```

Comment:
Honestly I was thinking the default `tib` for `simulate_hypno` should probably be 480 instead of 90, but not a big deal. More people deal with overnights than naps.

Reply:
Agreed. Changed!